syntax = "proto3";
package mongo;

message LoginCS
{
  string username = 1;
  string password = 2;
  int64 timestamp = 3;
}

enum LoginResult
{
  SUCCESS = 0;
  USERNAME_NOT_EXIST = 1;
  PASSWORD_NOT_CORRECT = 2;
}

message LoginSC
{
  LoginResult result = 1;
}